README.md

```diff
@@ -120,7 +120,7 @@ module "gitlab-runner" {
 | cache_bucket_prefix | Prefix for s3 cache bucket name. | string | `` | no |
 | cache_expiration_days | Number of days before cache objects expires. | string | `1` | no |
 | cache_shared | Enables cache sharing between runners, false by default. | string | `false` | no |
-| create_runners_iam_instance_profile |  | string | `true` | no |
+| create_runners_iam_instance_profile. |  | string | `true` | no |
```

Comment:
This overview is generated based on the variable.tf file. So changes here will be gone quite soon. I am still looking for a way to automate this.

Reply:
The '.' is a mistake I guess?